go.mod

```diff
@@ -1,16 +1,46 @@
 module github.com/stargate/stargate-grpc-go-client
 
-go 1.16
+go 1.18
 
 require (
 	github.com/docker/go-connections v0.4.0
 	github.com/google/uuid v1.2.0
-	github.com/opencontainers/runc v1.0.0-rc95 // indirect
 	github.com/sirupsen/logrus v1.8.1
 	github.com/stretchr/testify v1.7.0
 	github.com/testcontainers/testcontainers-go v0.11.1
 	google.golang.org/grpc v1.36.1
-	google.golang.org/grpc/cmd/protoc-gen-go-grpc v1.1.0 // indirect
 	google.golang.org/protobuf v1.27.1
 	gopkg.in/inf.v0 v0.9.1
 )
+
+require (
```

Comment:
Where did this `require()` section come from?

Reply:
Per https://go.dev/ref/mod#go-mod-file-require:

> At go 1.17 and above, the go command adds an indirect requirement for each module that provides any package imported (even [indirectly](https://go.dev/ref/mod#glos-indirect-dependency)) by a package or test in the main module or passed as an argument to go get. These more comprehensive requirements enable [module graph pruning](https://go.dev/ref/mod#graph-pruning) and [lazy module loading](https://go.dev/ref/mod#lazy-loading).

Reply:
(from running `go mod tidy`)

Reply:
Yeah, I figured, ran `go mod tidy` with `1.18. and was not able to replicate...That's why I asked.

Reply:
Thoughts?

https://user-images.githubusercontent.com/817672/169199153-6cb0436d-70e5-468d-ad8f-7be99e8acd16.mov